openedx/openedx-platform
# Remove surplus %20 and + from links to twitter and email

## Description

The course about page contains a link to Twitter, Facebook and Email in the sidebar on the right. These links lead to broken messages on Twitter and the email. For example, using `tutor`'s demo environment: 

https://demo.openedx.overhang.io/courses/course-v1:edX+DemoX+Demo_Course/about

If you click on the Twitter link on the right:

![image](https://user-images.githubusercontent.com/832723/106925517-72514b00-6710-11eb-9523-22a36e5bf5bb.png)

And if you click on the email button: 

![image](https://user-images.githubusercontent.com/832723/106925689-9a40ae80-6710-11eb-971c-db69d806b23e.png)

This bug impacts everybody that uses the default theme, or a theme based off it (like the [indigo theme](github.com/overhangio/indigo).

## Supporting information

I initially raised this problem in the [indigo repository](https://github.com/overhangio/indigo/issues/11)

## Testing instructions

To test this change, make sure your edX uses the default theme. Go to any course's about page and click the twitter icon. You should have no `+` signs in the twitter message. Now click the Email icon. You should have no `+` signs in the email icon.

## Deadline

None

## Other information

No other information I can think of
